pandas/core/internals/managers.py

```diff
@@ -327,24 +327,24 @@ def _verify_integrity(self) -> None:
                 f"tot_items: {tot_items}"
             )
 
-    def reduce(self, func, *args, **kwargs):
```

Comment:
can the edits in this file be done independent of the rest of the PR?  (also generic.py i think)